src/main/java/hu/oe/nik/szfmv/automatedcar/model/WorldObject.java

```diff
@@ -78,18 +116,27 @@ public void setHeight(int height) {
 
     /** {@inheritDoc}
      */
+    @Override
     public double getRotation(){
         return this.transform.getRotation();
     }
 
     /**
-     * @deprecated
+     * <b>Nem használható, csak Unit tesztelés miatt kell!<b/>
      * @param rotation rotation of the object
      */
-    public void setRotation(double rotation){
+    public void setRotation(double rotation) {
```

Comment:
ha nem akarjuk, hogy mások használják mehet package private-be talán (public nélkül)

Reply:
Ez egy jó ötlet lenne, de az XmlParserTest hívja meg, így nem opció sajnos. Szerintem ezt csak új konstruktorral lehetne megkerülni, ami várna egy Positont és egy Transformot. Ha van ötleted még, akkor jöhet!

Reply:
Majd refaktorban átnézzük ismét, amikor már felépült a rendszer